rust/tvm/examples/resnet/src/main.rs

```diff
@@ -76,7 +76,7 @@ fn main() {
         graph.into(),
         (&lib).into(),
         (&ctx.device_type).into(),
-        (&ctx.device_id).into(),
+        (&(ctx.device_id as i32)).into(),
```

Comment:
the real questions are
a) why this requires a reference
b) why `From<T> for DeviceId` isn't `impl`d for more types

anyway, thanks for catching this!